README.md

```diff
@@ -109,16 +116,6 @@ title="See more information on Transifex.com"><img src="https://ds0k0en9abmn1.cl
 </table>
 
 
-## Links
-
-* Website: [gitextensions.github.io](https://gitextensions.github.io/)
-* Source code: [github.com/gitextensions/gitextensions](https://github.com/gitextensions/gitextensions)
-* Online manual: [git-extensions-documentation.readthedocs.org](https://git-extensions-documentation.readthedocs.org/en/latest/)
-* Issue tracker: [github.com/gitextensions/gitextensions/issues](https://github.com/gitextensions/gitextensions/issues)
-* Wiki: [github.com/gitextensions/gitextensions/wiki](https://github.com/gitextensions/gitextensions/wiki)
-* Gitter chat: [gitter.im/gitextensions/gitextensions](https://gitter.im/gitextensions/gitextensions?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
-* Mailing list: [groups.google.com/group/gitextensions](https://groups.google.com/group/gitextensions)
-
 # Shoutouts
 
 * We thank JetBRAINS for [ReSharper](https://www.jetbrains.com/resharper/) licenses.
```

Comment:
A little tweak that I find could be done. Put contributors first, and after ReSharper and icons.

Reply:
Good idea, thank you